terraform/vpc/security-group.tf

```diff
@@ -14,14 +26,14 @@ resource "aws_security_group" "office-access-ssh" {
     from_port   = 22
     to_port     = 22
     protocol    = "tcp"
-    cidr_blocks = compact(concat(var.admin_cidrs, [var.concourse_egress_cidr]))
```

Comment:
You have removed the admin cidrs here which has caused us to loose access to bosh from our laptops.

Reply:
+1 to this suggestion

Reply:
Would you be ok with set_concourse_egress_cidrs? They are either being set or not which seems intuitive to me but then what do I know?